zc_common/remote_resource/filters.py

```diff
@@ -27,6 +29,14 @@ def filter_queryset(self, request, queryset, view):
                 if hasattr(queryset.model, field_name)\
                         and isinstance(getattr(queryset.model, field_name).field, ManyToManyField):
                     value = value.split(',')
+
+                # Allow 'true' or 'false' as values for boolean fields
+                try:
+                    if isinstance(queryset.model._meta.get_field(field_name), BooleanField):
+                        value = bool(strtobool(value))
+                except FieldDoesNotExist:
+                    pass
```

Comment:
What's the benefit of retrieving the value of the field in this way, using try/except and `_meta.get_field()`?

If it's better then the approach in the code above this, we should use it in both places.

If it's the same functionality though, I'd prefer the approach above, as it is smaller, and we know it works.